editor/osm_auth_tests/osm_auth_tests.cpp

```diff
@@ -42,6 +42,8 @@ UNIT_TEST(OSM_Auth_ForgotPassword)
   bool result;
   TEST_NO_THROW(result = auth.ResetPassword(kForgotPasswordEmail), ());
   TEST_EQUAL(result, true, ("Correct email"));
+
+  /// @todo Expect false? Anyway this function is not used.
   TEST_NO_THROW(result = auth.ResetPassword("not@registered.email"), ());
-  TEST_EQUAL(result, false, ("Incorrect email"));
+  TEST_EQUAL(result, true, ("Incorrect email"));
```

Comment:
1. It is true that `ResetPassword` is not used.
2. It is incorrect to change false to true.
3. A better option is to comment out this test and the corresponding function with TODOs: `// TODO: OSM API to reset password has changed and should be updated`